src/lib/classes/automaton/raw/RawQuery.ts

```diff
@@ -1,19 +1,16 @@
 import { z } from "zod";
-import { Backend } from "../Backend";
+//import { Backend } from "../Backend";
```

Comment:
Should be deleted if it is not used

Reply:
I honestly think it is better to keep it. It is only temporarily disabled because it will be re-added by a different PR. So we are keeping it here to show the old implementation until the new implementation is introduced. 

If you still think it is better to remove, I can do that as well.